test/data/ytcfg_mock_no_flags.html

```diff
@@ -0,0 +1,41 @@
+<html>
+<head>
+    <title>OK</title>
+    <script>
+    // Backup the original `replace` method.
+    const originalReplace = String.prototype.replace;
```

Comment:
there's a lot of duplication across these test files, I think it would be better to put them all in a shared js file (except for window.ytcfg). Can be fixed in a follow-up